palantirnet/federated-search-react
# Dependency security updates

Updates the following dependencies in order to upgrade vulnerable versions of `acorn` and `minimist`:
- eslint
- react-scripts
- node-sass-chokidar
- cypress

## Testing

- `yarn why acorn`
- `yarn why minimist`
- Verify the vulnerable versions are no longer present: see https://github.com/palantirnet/federated-search-react/network/alert/yarn.lock/acorn/open and https://github.com/palantirnet/federated-search-react/network/alert/yarn.lock/minimist/open
- Verify Federated Search still works as expected. Use demo site?
